wiki/Modding/es.md

```diff
@@ -4,76 +4,43 @@ outdated_translation: true
 
-Después de jugar y testear, mira la canción en el editor y pausa frecuentemente para buscar errores. En esta fase de modding es donde los modders encuentran errores mas instrincados como combo errors, inconsistencias de [hitsounds](/wiki/Beatmapping/Hitsound), y áreas donde los [hit circles](/wiki/Hit_object/Hit_circle) o [sliders](/wiki/Hit_object/Slider) son necesarios o innecesarios. Si encuentras que un hit circle o slider que necesita atención entonces seleccionalo, luego ve al tab “Edit” y selecciona “Copy” (esto también se puede hacer con Ctrl + C). Ahora ve a tu post (mod) que iniciaste anteriormente y pega lo que acabas de copiar (con Ctrl + V). Luego escribe a continuación porque crees tu que esta mal o que se puede cambiar/hacer sobre el hit circle o slider (ejemplo: “Yo cambiaría este espaciado porque la música no sugiere un [jump](/wiki/Mapping_techniques/Jumps) (salto) aquí), o simplemente que el circle o slider no se siente bien al hacer click sobre el. También busca por cosas de las que no te percataste al jugar el map (Por ejemplo archivos innecesarios en la carpeta).
+- Los modders reconocidos a menudo anotan en sus perfiles de usuario si están o no aceptando solicitudes de mods.
 
-**Repite lo mismo con la siguiente dificultad**, ten en mente que si ves algo en el editor de lo que no te diste cuenta al jugar, señalalo también. (Es recomendado que también reproduzcas el map a velocidad normal en el editor para corroborar que todo este bien). Cuando estés listo has submit a tu post.
+- Buscando el listado de beatmaps pendientes como beatmaps para modear **no** son recomendados debido a que la mayoría de envíos no están apuntando al estado Ranked, por lo que sus creadores no responderán potencialmente por mods.
 
-## ¿Qué debes mirar mientras modeas un mapa?
+## Como modear
 
-Una herramienta muy útil mientras es el AImod, el cuál puede ser abierto debajo de la barra de herramientas del editor o presionando Ctrl+Shift+A. Esto muestra los errores que la dificultad actual tiene del betmap. Sin embargo, ten en cuenta que es sólo un programa y algunos errores no son verdaderos errores. (Por ejemplo: un error de espaciado, cuando se suponía que debía haber un salto o jump)
+Para modear un beatmap, empieza descargando la versión más reciente y abre su [página de discusión](/wiki/Beatmap_discussion). Juega la [dificultad](/wiki/Beatmap/Difficulty) que quieras modear y ten en cuenta cualquier problema relacionado con el gameplay. Cuando finalices, abre la dificultad en el editor y analízala para encontrar otros errores.
 
-La siguiente lista está organizada según su importancia en el beatmap.
+Cuando te encuentres con un problema, selecciona los hit objects que sean relevantes, luego usa el atajo de copia (`Ctrl` + `C`) para copiar la información de la línea de tiempo como texto. En la discusión del beatmap, pega (`Ctrl` + `V`) cualquier línea de tiempo aplicable y tu retroalimentación.
```

Comment:
**Missing all this sentance _(see en.md file)_:**
The `Timeline` tab is used for issues regarding specific parts of a beatmap, the `General (This difficulty)` tab is used for feedback on larger sections of a beatmap, and the `General (All difficulties)` tab is used for issues applicable to all difficulties of a beatmap.

Reply:
Added.